Tasks/google-play-release/GooglePlay.ts

```diff
@@ -740,6 +854,69 @@ function resolveGlobPath(path: string): string {
     return path;
 }
 
+/**
+ * Get the appropriate files from the provided pattern
+ * @param {string} path The minimatch pattern of glob to be resolved to file path
+ * @returns {string[]} paths of the files resolved by glob
+ */
+function resolveGlobPaths(path: string): string[] {
+    if (path) {
+        // Convert the path pattern to a rooted one. We do this to mimic for string inputs the behaviour of filePath inputs provided by Build Agent.
+        path = tl.resolve(tl.getVariable('System.DefaultWorkingDirectory'), path);
```

Comment:
Does this only work for relative paths?  What if someone specifies an absolute path like:
`$(Build.ArtifactStagingDirectory)/target/**/MyApp.apk`
It appears this code would translate that into:
`$(System.DefaultWorkingDirectory)/$(Build.ArtifactStagingDirectory)/target/**/MyApp.apk`